wombase_backend/server/apps/staff/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from .models import Employee
+
+
+class EmployeeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Employee
+        fields = '__all__'
```

Comment:
specify only necessary fields. For example, password field is completely redundant here. Also, fields like is_staff, is_superuser, etc are unnecessary for frontend interaction with api